### ISSUE TITLE
Bug Report: [DESCRIBE YOUR BUG IN DETAIL HERE]

### ISSUE THREAD

**Issue by pizzapwner** [YoshiCrafter29/YoshiCrafterEngine#348]
## Which target are you using?
- [x] Windows
- [ ] Mac
- [ ] Linux

## What version of the engine are you using (Bottom left of the screen in main menu) ?
2.3.2

## Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.
Basically, if a character is selected as a player, if the spritesheet is the wrong way flipped, singLEFT and singRIGHT for one slight frame, goes to another position.
## Please describe your issue. Provide extensive detail and images if possible.
Characters spritesheet (example)
![image](https://user-images.githubusercontent.com/110793808/192659797-0652f34c-06a0-4ea3-95ee-a62b722a5fe0.png)
Character in-game:
![show](https://user-images.githubusercontent.com/110793808/192659915-2ad241bf-78e2-4dc7-912c-15f7b03325f4.gif)
---

**Comment by SpaceVeta** [comment]
Yeah this is a issue me and a friend have also been having. Please address if there's a solution to this.

**Comment by SIG7Pro** [comment]
That may just be a problem with the spritesheet and/or the xml file.
